app/views/users/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<main class="page">
+<header class="navbar">
+    <div>
+        <%= render 'shared/eventbrite_logo'%>
+    </div>
+
+    <div>
+        <%= link_to "Home", root_path, class: "eb-btn"%>
+    <% if logged_in? %>
+        <%= link_to "Logout", logout_path, method: :delete, class: "eb-btn"%>
+    <% end %>
+    </div>
+</header>
+<div class="row m-0">
+<div class="col-6 card p-5 m-5">
+<h1>My profile</h1>
+<br>
+<h1><%= @user.name %></h1>
+<h4><%= @user.email %></h4>
+</div>
+</div>
+
+</main>
+
+<section class="container m-5">
+<h2 class="m-4">Events</h2>
```

Comment:
The User show page should have events created by the user *and* events the user is attending separated into Upcoming and Previous events. We can only see upcoming and previous here but there isn't any distinction between events created by the user from events the user is attending.

[`REQUIRED`]